time.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./assets/css/bootstrap.min.css">
+    <link rel="stylesheet" href="./assets/css/style.min.css">
+</head>
+<body>
+    <div class="container-fluid p-0"></div>
+    <script>
+        fetch('./base.html')
```

Comment:
точка не нужна, вообще относительные пути зло, везде бы поправить
```suggestion
        fetch('/base.html')
```

Reply:
поправлено